### ISSUE TITLE
Can't use this package in react native 0.61.5 ?

### ISSUE THREAD

**Issue by tscj3490** [viromedia/viro#865]
I could run app in react-native 0.59, but in 0.61.5, app is crashed. 
Please upgrade version so that developers can use this package in react-native 0.60 + versions.
Thanks

**Comment by jpudysz** [comment]
You can but with this line inside package.json
`"react-viro": "npm:@iskander508/react-viro@^0.61.2"`

**Comment by ranjanpoudel1234** [comment]
@tscj3490 you good on this ? If so, could you close the issue ?

**Comment by tscj3490** [comment]
No, I am still struggling.. Would you provide a sample repo working in react native 0.61.5 or more version ?

**Comment by ranjanpoudel1234** [comment]
I am using this in my package.json https://github.com/mendix/viro, can you try this source directly for react-viro in your package.json ?

**Comment by tscj3490** [comment]
oki.. let me try with it.

**Comment by greenbaypayment** [comment]
> I am using this in my package.json https://github.com/mendix/viro, can you try this source directly for react-viro in your package.json ?

For me, it doesn't work. What react native version do u use ?

**Comment by Turtleted21** [comment]
this resolve the ios 13.5 bug UIWebView ?

**Comment by tscj3490** [comment]
> this resolve the ios 13.5 bug UIWebView ? VR work after update?

No, I can't still deploy viro app to apple store.. Apple doesn't approve.

**Comment by Akash76** [comment]
I am able to use viro in RN 0.63.3 by doing following:

1. Clone viro repository
2. Updated react and react-native versions to latest version in package.json
3. Updated react and react-native versions to latest version in package.json in test folder and run npm install (May be not necessary)
4. execute prepare_release.sh in root directory => It generates react-viro-2.17.0.tgz file
5. Created project with react-native init and installed react-viro and create-react-class using npm
6. Remove all the contents of react-viro in node-modules and then extract the contents of tgz file into cleared react-viro.
7. execute this script to make all necessary modifications in gradle, AndroidManifest.xml and java files: ./node_modules/react-viro/bin/android-setup.sh <project-name> true. Modified few errors
8. Then running android script with GvrDebug, react-native run-android --variant=GvrDebug

The above mentioned steps made my VR app to run perfectly.
refer this gist to see all the necessary changes: https://gist.github.com/akeemphilbert/8323a7b71e682469b922920a624b35a8

Hope this helps

**Comment by SpQuyt** [comment]
> I am able to use viro in RN 0.63.3 by doing following:
> 
> 1. Clone viro repository
> 2. Updated react and react-native versions to latest version in package.json
> 3. Updated react and react-native versions to latest version in package.json in test folder and run npm install (May be not necessary)
> 4. execute prepare_release.sh in root directory => It generates react-viro-2.17.0.tgz file
> 5. Created project with react-native init and installed react-viro and create-react-class using npm
> 6. Remove all the contents of react-viro in node-modules and then extract the contents of tgz file into cleared react-viro.
> 7. execute this script to make all necessary modifications in gradle, AndroidManifest.xml and java files: ./node_modules/react-viro/bin/android-setup.sh  true. Modified few errors
> 8. Then running android script with GvrDebug, react-native run-android --variant=GvrDebug
> 
> The above mentioned steps made my VR app to run perfectly.
> refer this gist to see all the necessary changes: https://gist.github.com/akeemphilbert/8323a7b71e682469b922920a624b35a8
> 
> Hope this helps

Excuse me!
Have you test this on iOS devices?
